shared/elements/gaia_lowstorage_dialog/examples/locales/example.en-US.properties

```diff
@@ -0,0 +1,7 @@
+lowstorage-dialog-title=Low Device Storage
```

Comment:
I asked myself this too; maybe I should split it already and let the app developers decide.